### ISSUE TITLE
List of sketchy domains

### ISSUE THREAD

**Issue by thisisu** [StevenBlack/hosts#486]
Hey there,

I'm new to GitHub and like your project. I want to contribute so I created https://github.com/thisisu/hosts/blob/master/hosts. Feel free to browse for any sketchy domains you might be missing.

Cheers

**Comment by StevenBlack** [comment]
Thanks @thisisu!

One of my criteria for active inclusion is active, ongoing curation of the data.

This list amalgamates several sources, some of which have been actively curated for many years; decades in some cases.

The repo https://github.com/thisisu/hosts is very new.  Are you committed to keeping this updated?

Also, where are you getting these domains?  What's your process?

**Comment by thisisu** [comment]
Hi @StevenBlack 

Thanks for the information. Unfortunately, I don't think I meet your commitment requirements. This is just a hobby for me, as I think it is with you and others, however I must admit I'm not much of a fan of commitment. If you want some actual data, I've updated the repo 45 times since its original commit 8 days ago.
Most of the domains were gathered and added into AdAway's custom block list while I was surfing porn on an Android phone. =) Since then, I've just bookmarked some of the known bad redirects on a Windows 7 VM and just constantly refresh them when I'm looking for new domains to add to hosts' blacklist.
